scripts/dcos-windows-tox-tests.ps1

```diff
@@ -0,0 +1,43 @@
+$ErrorActionPreference = "Stop"
+
+
+function Confirm-DCOSToxRequirements {
+    # - Make sure the correct Python version is installed
+    $pythonVersion = python.exe --version
+    if($LASTEXITCODE) {
+        Throw "Failed to get the Python version"
+    }
+    if($pythonVersion -ne "Python 3.6.5") {
+        Throw "Python has wrong version! Expected at least 3.6.5 and found: $pythonVersion"
```

Comment:
The phrasing of the error message could be confusing.
The test looks for a hard match of "Python 3.6.5", but the message implies that newer versions would succeed, though the logic implies they would not.
Something like "Expected: $expectedPythonVersion but found: $pythonVersion"